File: javascript_objects_scopes_closures/101-sorted.js
#!/usr/bin/node

const { dict } = require('./101-data');

const newDict = {};

for (const [key, value] of Object.entries(dict)) {
  const strValue = value.toString();
  if (!(strValue in newDict)) {
    newDict[strValue] = [];
  }
  newDict[strValue].push(key.toString());
}

File: javascript_objects_scopes_closures/101-sorted.py
#!/usr/bin/python3

from data_101 import content

new_dict = {}

for key, value in content.items():
    str_val = str(value)
    str_key = str(key)
    if str_val not in new_dict:
        new_dict[str_val] = []
    new_dict[str_val].append(str_key)

print(new_dict)
